vendor/index.yaml

```diff
@@ -2052,3 +2052,11 @@ vendors:
     description: Lansitec is a technology-driven company providing cutting-edge active people and asset tracking products and services. i.e., trackers, gateways and bridges, beacons and tags, and related accessories like sensors.
     logo: lansitec.png
     website: https://www.lansitec.com/
+
+  - id: redz
+    name: REDZ Smart Communication
+    description: REDZ Smart Communication Technologies is a provider for industrial coomunication devices such as LoraWAN EndNode Modems, Electricity Meter Protocol to Modbus Protocol Gateways, LoRa RF Modems, Wireless Mbus Modems, Remote IO Devices, Serial to ETH Gateways, Industrial Unmanaged Switches with or without Broadband Powerline (BPL) connection and Energy Meter Reading Optical Probes.
+    logo: redz.jpg
```

Comment:
@ali-ugur It seems you forgot to run our validation: https://github.com/TheThingsNetwork/lorawan-devices?tab=readme-ov-file#validation

Im getting an error that the logo should be either png or svg